shared/libraries/config_protocol/include/config_protocol/config_protocol.h

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2022-2023 Blueberry d.o.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <functional>
+#include <stdexcept>
+#include <string>
+#include <coretypes/string_ptr.h>
+
+namespace daq::config_protocol
+{
+
+enum PacketType: uint8_t { getProtocolInfo = 0x80, upgradeProtocol = 0x81, rpc = 0x82, serverNotification = 0x83, invalidRequest = 0x84 };
```

Comment:
Would it make sense to have these be a few bits apart? Eg. 0x80, 0x85, 0x90..?

Reply:
I actually don't expect many more packet types. Almost all communication is either through rpc or notification packet type.